go/ql/test/library-tests/semmle/go/frameworks/GoMicro/main.go

```diff
@@ -0,0 +1,39 @@
+package main
+
+//go:generate depstubber -vendor go-micro.dev/v4 Service,Option,Options NewService,Name,Handle,Server,Client
+//go:generate depstubber -vendor go-micro.dev/v4/server Server Handle
+//go:generate depstubber -vendor go-micro.dev/v4/client Client Call
+
+import (
+	pb "codeql-go-tests/frameworks/GoMicro/proto"
+	"context"
+	"fmt"
+	"log"
+
+	micro "go-micro.dev/v4"
+)
+
+type Greeter struct{}
+
+func (g *Greeter) Hello(ctx context.Context, req *pb.Request, rsp *pb.Response) error {
```

Comment:
Switch to inline expectations test
```suggestion
func (g *Greeter) Hello(ctx context.Context, req *pb.Request, rsp *pb.Response) error { // $ serverRequest="definition of req"
```